rskj-core/src/main/resources/reference.conf

```diff
@@ -350,6 +350,8 @@ sync {
             limit = 1000000
             # Parallel requests (true, false
             parallel = false
+            # list of SNAP-capable peers to connect to
+            trustedPeers = []
```

Comment:
I think we should initiate connections to peers listed in this array in the `NodeManager`'s  ctor, same as it's done for active peers in [here](https://github.com/rsksmart/rskj/blob/b665c02a07faf8053b21da2e62e63c0ff1978e28/rskj-core/src/main/java/org/ethereum/net/NodeManager.java#L60-L64), wdyt?

also, as for me, the `trustedPeers` name "collides" a bit with another config `peer.trusted`, which defines peers that are allowed to connect to this node. What do you think about naming it something like `active` or `bootNodes`?

Reply:
addressed, changed the name to snapBootNodes and initiated connections in the NodeManager's constructor. Check [here](https://github.com/rsksmart/rskj/pull/2504/files#diff-a595dadc02034651fb8d68489f41bbfeedf97fdf4e9682d8056c34fbaf95b8a5R65-R71)